Content.Server/Nutrition/Components/HungerComponent.cs

```diff
@@ -22,24 +23,22 @@ public sealed class HungerComponent : SharedHungerComponent
     {
         [Dependency] private readonly IRobustRandom _random = default!;
 
-        [DataField("base_decay_rate")]
-        private float _baseDecayRate = 0.1f;
+        // TODO QUESTION This was previously dealing blunt damage. this doesn't feel right, but I will leave it here.
+        // Same thing for ThirstComponent
+        // TODO PROTOTYPE Replace this datafield variable with prototype references, once they are supported.
+        [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+        [DataField("damageType")]
+        private readonly string _damageTypeID = "Blunt";
+        private DamageTypePrototype _damageType => _prototypeManager.Index<DamageTypePrototype>(_damageTypeID);
 
-        [DataField("damageType",required: true)]
-        private readonly DamageTypePrototype _damageType = default!;
+        private float _acumulatedDamage;
 
-        private float _actualDecayRate;
-        private float _currentHunger;
-        private HungerThreshold _currentHungerThreshold;
-        private HungerThreshold _lastHungerThreshold;
-        private readonly Dictionary<HungerThreshold, float> _hungerThresholds = new()
-        {
-            {HungerThreshold.Overfed, 600.0f},
-            {HungerThreshold.Okay, 450.0f},
-            {HungerThreshold.Peckish, 300.0f},
-            {HungerThreshold.Starving, 150.0f},
-            {HungerThreshold.Dead, 0.0f},
-        };
+        // TODO QUESTION Just based on DrSmugleaf's other comments on similar situations: are all of _baseDecayRate,
```

Comment:
In the other PR, they pointed out some variables were redundant and should be removed. Is there some reason these are different?

Reply:
i'm guessing 'base decay rate' is meant to be the initial decay rate and 'actual decay rate'  is how fast its actually decaying, independent of that? i would just remove 'actual decay rate'

Reply:
The actualDecayRate is either 1.2*, 1*, or 0.8* baseDecayRate, depending on whether they are overfed, normal, or starving. The question about redundancy was about whether _baseDecayRate should just be replaced with BaseDecayRate, and similarly for all the others.